### PR TITLE
Add support for Swedish holidays

### PR DESCRIPTION
**Pull request by adevade**
There's already a [pull request open for Sweden](https://github.com/spatie/holidays/pull/137). This PR fixes the requested changes.

### CODE REVIEW

File: src/Countries/Sweden.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            'Juldagen' => '12-25',
+            'Annandag jul' => '12-26',
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+        $easter = $this->easter($year)->setTimezone('Europe/Stockholm');
+
```
Comment (Nielsvanpach) [inline src/Countries/Sweden.php:30]:
```suggestion
        $easter = $this->easter($year);

```

Comment (Nielsvanpach) [reply src/Countries/Sweden.php:30]: Timezones shouldn't be needed

File: src/Countries/Sweden.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        /** @var CarbonImmutable $midsummerDay */
+        $midsummerDay = CarbonImmutable::createFromFormat('Y-m-d', "{$year}-06-20", 'Europe/Stockholm');
```
Comment (Nielsvanpach) [inline src/Countries/Sweden.php:32]: Timezones shouldn't be needed. You can call ->startOfDay() on the Carbon instance.

Example: https://github.com/spatie/holidays/blob/main/src/Countries/Country.php#L52

File: src/Countries/Sweden.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+        if (! $midsummerDay->isSaturday()) {
+            $midsummerDay = $midsummerDay->next(CarbonImmutable::SATURDAY);
+        }
+
+        /** @var CarbonImmutable $halloween */
+        $halloween = CarbonImmutable::createFromFormat('Y-m-d', "{$year}-10-31", 'Europe/Stockholm');
```
Comment (Nielsvanpach) [inline src/Countries/Sweden.php:39]: Same comment as above

**Review by Nielsvanpach — CHANGES_REQUESTED** [review]
Thanks! In case you need translations: https://github.com/spatie/holidays/tree/main/lang

File: src/Countries/Sweden.php (hunk 2 above)
Comment (adevade) [reply src/Countries/Sweden.php]: I saw that some countries added timezones, and some do not. 

https://github.com/spatie/holidays/blob/914ae68e4dd979172182fb60058ae914db12c0f8/src/Countries/Australia.php#L33

Comment (Nielsvanpach) [reply src/Countries/Sweden.php]: Thanks, I fixed some of them in the meantime.

**Review by Nielsvanpach — APPROVED** [review]
Thanks!

**Pull request merged**